testdata/output/telex__simple_telex.snap

```diff
@@ -67,4 +67,9 @@ trường
 ươi
 cười
 hường
+chửa
+chuă
```

Comment:
This behaves the same before this PR.

Reply:
Yeah this one is a bit weird. But I guess it make sense 😂 . There's not much benefit in trying to come up with a proper way to handle this case since I don't think people will run into this situation often. I think this is fine.